### DISCUSSION TITLE
How do you add a value constraint to a column?

### DISCUSSION THREAD

**Discussion by quantdrogon** [prisma/prisma#13526] [Q&A] [answered]
Suppose I want a column in a table to only have int's in the range [0,5], i.e. {0,1,2,3,4,5}, how do I specify that in the prisma schema file?

Would use this to implement ratings in my database.

**Comment by nurul3101** [comment, marked as answer]
Hey @quantdrogon 👋 

You could enforce this by enums and also by a check constraint. You could add a constraint in your database like this
```
alter table ratings
   add constraint check_rating
   check (rating between 0 and 5);
```  
You won’t need to define this in schema file, you can define the rating column in your model file as Int and enforce the above constraint on your database, entering any other values except [0..5] will throw an error.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
